chroma-agent/chroma_agent/device_plugins/lustre.py

```diff
@@ -121,32 +152,40 @@ def reset_state(self):
     def _scan_mounts(self):
         mounts = {}
 
-        for device, mntpnt, fstype in Mounts().all():
+        data = scanner_cmd("Stream")
+        local_mounts = get_local_mounts(data['localMounts'])
+        zfs_mounts = [(d, m, f) for d, m, f in local_mounts if f == 'zfs']
+
+        for device, mntpnt, fstype in local_mounts:
             if fstype != 'lustre':
```

Comment:
nit: Like you have `zfs_mounts` above you can do:

lustre_mounts = [x for x in local_mounts if f == 'lustre']